lineapy/graph_reader/apis.py

```diff
@@ -114,9 +115,11 @@ def get_code(self, use_lineapy_serialization=True) -> str:
         track(
             GetCodeEvent(use_lineapy_serialization=True, is_session_code=False)
         )
-        return self._de_linealize_code(
-            str(get_source_code_from_graph(self._get_subgraph())),
-            use_lineapy_serialization,
+        return prettify(
```

Comment:
this prettify should not be required. the str(get_source_code_from_graph....) should return prettified code as the changes in program_slice.py indicate.

Reply:
hmm okay. we might reconsider using regex inside de_linealize and use the graph directly.. we can simplify this then.